src/Sum Root to Leaf Numbers.js

```diff
@@ -0,0 +1,40 @@
+/**
+ * Definition for a binary tree node.
+ * function TreeNode(val, left, right) {
+ *     this.val = (val===undefined ? 0 : val)
+ *     this.left = (left===undefined ? null : left)
+ *     this.right = (right===undefined ? null : right)
+ * }
+ */
+/**
+ * @param {TreeNode} root
+ * @return {number}
+ */
+
+var sumNumbers = function (root) {
+  const result = [];
+
+  function recursion(root, rootToLeaf = "") {
+    rootToLeaf += root.val;
+
+    if (!root.left && !root.right) return rootToLeaf;
+
+    if (root.left) {
+      result.push(recursion(root.left, rootToLeaf));
+    }
+
+    if (root.right) {
+      result.push(recursion(root.right, rootToLeaf));
+    }
```

Comment:
```suggestion
    if (root.left) result.push(recursion(root.left, rootToLeaf));
   
    if (root.right) result.push(recursion(root.right, rootToLeaf));
```
이 부분도 위에 20번째 줄과 통일성있게 작성할 수 있을 꺼 같습니다!